pkg/controller/nodeipam/ipam/range_allocator_test.go

```diff
@@ -292,13 +292,6 @@ func TestOccupyPreExistingCIDR(t *testing.T) {
 }
 
 func TestAllocateOrOccupyCIDRSuccess(t *testing.T) {
-	// Non-parallel test (overrides global var)
-	oldNodePollInterval := nodePollInterval
-	nodePollInterval = test.NodePollInterval
```

Comment:
this is just using 10 ms,  if you remove it , it will use 10 second,

Reply:
I don't think that's the case here, because `wait.PollUntilContextTimeout` have `immediate` arg set. That means we are sending request to `FakeNodeHandler` immediately instead of waiting for the first poll (no matter if it's set to 10ms or 10s).

In description I have attached before/after results for `k8s.io/kubernetes/pkg/controller/nodeipam/ipam` – the time is slightly lower, but definitely tests don't wait 10 seconds to poll.

Reply:
but this assumes that immediate always succeeds, no? if it has to retry it will be longer

So why don't just change only the PollUntilContextTimeout with inmediate?

Reply:
Yes, it assumes we will succeed at the first attempt in tests because we are using `FakeNodeHandler`. I don't see a reason why it would poll again in these particular tests.

Of course, I can revert these changes, though I think it can cause some confusion in the future as to why it is there when it doesn't really change the timing or outcome of the test.

Reply:
CI is more agressive environments and race happens causing flakes, I didn't checked the code myself and I can see if there can be a situation the poll can race and then have to retry, but I don't see any reason why moving to PollUntilContextTimeout to immediate must be done at the same time we change this variable

Reply:
Alright, I didn't think this way, just wanted to make code simpler. I'll revert this part.